scripts/SDM/rangePredictions.yml

```diff
@@ -16,3 +16,12 @@ outputs:
     label: range predictions
     description: range of a 95% confidence interval of a set of predictions
     type: image/tiff;application=geotiff
+conda:
+  channels:
+    - conda-forge
+    - r
+  dependencies:
+    - r-terra
+    - r-rjson
+    - r-raster
```

Comment:
Just a comment : using both terra and raster? We should probably fix that in a future PR...